charts/orchestrator-k8s/README.md

```diff
@@ -0,0 +1,61 @@
+# Orchestrator-k8s helm chart 
+
+This chart will install the Orchestrator and all its dependencies on kubernetes. 
+It is well suited for dev/QE/test or resource constrained environments.
+
+The chart deploys:
+- Janus-IDP
+- Serverless Workflows Operator (see sonata-serverless-operator.yaml)
+- knative serving
+- Knative eventing
+- Serverless Workflows (optional)
+
+### Usage
+
+```console
+helm repo add orchestrator https://parodos-dev.github.io/orchestrator-helm-chart
+
+helm install orchestrator orchestrator-k8s
+```
+
+
+## Development
+```console
+git clone https://github.com/parodos-dev.github.io/orchestrator-helm-chart
+
+cd orchestrator-helm-chart/charts/orchestrator-k8s
+
+helm dependencies build
```

Comment:
should this be `build` or `update`?

Reply:
for a clean directory it should be build. I want to exclude the charts/*.tgz from git